docs/config/README.md

```diff
@@ -65,6 +65,7 @@ This is the list of prompt-wide configuration options.
 | -------------- | ----------------------------- | ------------------------------------------------------ |
 | `add_newline`  | `true`                        | Add a new line before the start of the prompt.         |
 | `prompt_order` | [link](#default-prompt-order) | Configure the order in which the prompt module occurs. |
+| `scan_timeout` | `30`                          | Timeout for starship to scan files (in milliseconds).  |
```

Comment:
30 ms seems like a good default. 👍 
Testing locally on my 2.2 GHz i7 Macbook Pro 2018, the starship project was able to finish scanning in ~300µs, while a directory with 10,000 files took 21ms.